spyder/plugins/completion/languageserver/plugin.py

```diff
@@ -52,14 +52,7 @@ def __init__(self, parent):
         self.requests = set({})
         self.register_queue = {}
 
-        # Register languages to create clients for
-        for language in self.get_languages():
-            self.clients[language] = {
-                'status': self.STOPPED,
-                'config': self.get_language_config(language),
-                'instance': None
-            }
-            self.register_queue[language] = []
```

Comment:
I don't think replacing these lines by `update_configuration` is the right thing to do here. I mean, here we're only creating the list of client to start clients for, which is done later. However, `udpate_configuration` tries to restart clients too.

@andfoy, what do you say?

Reply:
I think this is valid because we're performing the same initialization on ``update_configuration``